src/lib/tooltip/tooltip.spec.ts

```diff
@@ -432,6 +432,15 @@ describe('MdTooltip', () => {
       flushMicrotasks();
       expect(tooltipDirective._tooltipInstance).toBeNull();
     }));
+
+    it('should have rendered the tooltip text on init', fakeAsync(() => {
+      dispatchFakeEvent(buttonElement, 'mouseenter');
+      fixture.detectChanges();
+      tick(0);
+
+      const tooltipElement = overlayContainerElement.querySelector('.mat-tooltip') as HTMLElement;
+      expect(tooltipElement.textContent).toContain('initial tooltip message');
+    }));
```

Comment:
It already is using the one that we use for testing `OnPush`: https://github.com/angular/material2/pull/3671/files#diff-90d01e051e22bab8526ca923091f589fR391